arch/arm/boot/dts/overlays/Makefile

```diff
@@ -19,6 +19,7 @@ dtbo-$(RPI_DT_OVERLAYS) += akkordion-iqdacplus.dtbo
 dtbo-$(RPI_DT_OVERLAYS) += allo-piano-dac-pcm512x-audio.dtbo
 dtbo-$(RPI_DT_OVERLAYS) += at86rf233.dtbo
 dtbo-$(RPI_DT_OVERLAYS) += audioinjector-wm8731-audio.dtbo
+dtbo-$(RPI_DT_OVERLAYS) += audioinjector-addons.dtbo
```

Comment:
I know it's picky, but can you put this in front of the previous line (in lexicographic order).